backend/siarnaq/bracket/challonge.py

```diff
@@ -25,7 +25,7 @@
 }
 
 AUTH_TYPE = "v1"
```

Comment:
What's this?

Reply:
Challonge's _authentication_ version
v1 is with keys, v2 is the weird OAuth/MFA stuff that we couldn't figure out last year

Reply:
seems unused

Reply:
huh good catch